### PR TITLE
{bio}[foss/2017b] TopHat v2.1.1

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.8, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/6d4d9db695e339b4a4695a904f60f922 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/46351ff012a9c9e1eb40e6f60923cd6a for a full test report.

File: easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-foss-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+name = 'TopHat'
+version = '2.1.1'
+
+homepage = 'http://ccb.jhu.edu/software/tophat/'
+description = """TopHat is a fast splice junction mapper for RNA-Seq reads."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://ccb.jhu.edu/software/tophat/downloads/']
```
Comment (verdurin) [inline easybuild/easyconfigs/t/TopHat/TopHat-2.1.1-foss-2017b.eb]: @vanzod Better to use `%(namelower)s` where appropriate here?

**Comment by verdurin** [comment]
Going in, thanks @vanzod!

**Pull request merged**